### PR TITLE
Add ru.toxblh.KeeneticManager

### PR DESCRIPTION
**Pull request by Toxblh**
### Please confirm your submission meets all the criteria

- [x] Please describe the application briefly.
Keenetic Manager: Provides the ability to manage some features of the Keenetic router more easily

- [x] The domain used for the application ID is [controlled by the application developers][appid-domain] and the [application id guidelines][appid] are followed.
- [x] I have read the and followed all the [Submission and licence requirements][reqs].
- [x] I have [built][build] and tested the submission locally.
- [x] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about this submission. **Link:**

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements/#control-over-domain-or-repository
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

File: ru.toxblh.KeeneticManager.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        "--socket=wayland",
+        "--filesystem=xdg-pictures",
+        "--talk-name=org.freedesktop.Notifications",
+        "--talk-name=org.freedesktop.secrets",
+        "--talk-name=org.gnome.SettingsDaemon.Color"
+    ],
+    "build-options": {
+        "append-path": "/usr/lib/sdk/rust-stable/bin",
+        "build-args": [
+            "--share=network"
```
Comment (hfiguiere) [inline ru.toxblh.KeeneticManager.json:24]: this is not allowed (and it will fail). Build must be offline.

File: ru.toxblh.KeeneticManager.json [hunk 2]
```diff
@@ -0,0 +1,60 @@
+{
+    "id" : "ru.toxblh.KeeneticManager",
```
Comment (bbhtt) [inline ru.toxblh.KeeneticManager.json:2]: can you link your github profile somewhere on https://toxblh.com/?

File: ru.toxblh.KeeneticManager.json [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    ],
+    "command" : "keeneticmanager",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--filesystem=xdg-pictures",
+        "--talk-name=org.freedesktop.Notifications",
```
Comment (bbhtt) [inline ru.toxblh.KeeneticManager.json:17]: is notifications needed? It should use the portal.

**Comment by bbhtt** [comment]
Does this have English localization? If not please add a line in description saying it doesn't https://github.com/Toxblh/Keenetic-Manager/blob/c35177e98d573f6ec8b1a58b71e7a5e5968fecf9/data/ru.toxblh.KeeneticManager.metainfo.xml.in#L10

File: ru.toxblh.KeeneticManager.json (hunk 2 above)
Comment (Toxblh) [reply ru.toxblh.KeeneticManager.json:2]: Yeah that linked here: https://toxblh.com/contacts/

Comment (hfiguiere) [reply ru.toxblh.KeeneticManager.json:2]: but the id is `ru.toxblh`.... not `com.*`

Comment (Toxblh) [reply ru.toxblh.KeeneticManager.json:2]: https://toxblh.ru/contacts/ both is mine

File: ru.toxblh.KeeneticManager.json (hunk 3 above)
Comment (Toxblh) [reply ru.toxblh.KeeneticManager.json]: no, removed

File: ru.toxblh.KeeneticManager.json (hunk 1 above)
Comment (Toxblh) [reply ru.toxblh.KeeneticManager.json]: that fixed

File: ru.toxblh.KeeneticManager.json (hunk 2 above)
Comment (Toxblh) [reply ru.toxblh.KeeneticManager.json:2]: But I would like to use ru. version.

File: ru.toxblh.KeeneticManager.json (hunk 3 above)
Comment (Toxblh) [reply ru.toxblh.KeeneticManager.json]: that fixed as well

**Comment by Toxblh** [comment]
> Does this have English localization? If not please add a line in description saying it doesn't https://github.com/Toxblh/Keenetic-Manager/blob/c35177e98d573f6ec8b1a58b71e7a5e5968fecf9/data/ru.toxblh.KeeneticManager.metainfo.xml.in#L10

English added and that is main lang now, others will in /po

**Comment by bbhtt** [comment]
bot, build ru.toxblh.KeeneticManager

**Comment by Toxblh** [comment]
bot, build ru.toxblh.KeeneticManager

**Comment by Toxblh** [comment]
bot, build ru.toxblh.KeeneticManager

**Comment by Toxblh** [comment]
Looks like finaly ready :)

**Pull request closed without merging**